frontend/app/(footer)/event/create/page.tsx

```diff
@@ -108,7 +109,15 @@ export default function CreateEvent() {
       "https://firebasestorage.googleapis.com/v0/b/socialsports-44162.appspot.com/o/users%2Fgeneric%2Fgeneric-sports.jpeg?alt=media&token=045e6ecd-8ca7-4c18-a136-71e4aab7aaa5";
 
     if (formData.image !== undefined) {
-      imageUrl = await uploadUserImage(user.userId, formData.image);
+      const originalFile = formData.image;
+      const modifiedFileName = uuidv4();
+
+      const modifiedFile = new File([originalFile], modifiedFileName, {
```

Comment:
we probably want the this function in the uploadUserImage function, rather than out here in the page.tsx. 

This is logic that should be abstracted away, and any further usage of uploadUserImage should have this functionality inherently coupled in to the upload of an image.

Reply:
ah sorry yes ur right